src/__tests__/ceramic_integration.test.ts

```diff
@@ -425,6 +436,16 @@ describe('Ceramic Integration Test', () => {
         const doc2 = await TileDocument.load(ceramic2, doc1.id)
         await doc2.update(updatedContent, null, { anchor: false })
 
+        // Make sure that the ceramic CAS has received the newest version
+        const casDocRef = await casCeramic1.loadStream(doc1.id)
+        await firstValueFrom(
+          casDocRef.pipe(
+            filter((state) => {
+              return state.next?.content.foo === updatedContent.foo
+            })
+          )
+        )
+
         await anchorUpdate(doc1, cas1)
 
         expect(doc1.state.anchorStatus).toEqual(AnchorStatus.ANCHORED)
```

Comment:
I think the checks on lines 454-456 are subject to the same problem where the pubsub message may just not have made it yet.

Reply:
ping.  Can we fix that one the same way?

Reply:
yup sorry got confused